adafruit_bus_device/i2c_device.py

```diff
@@ -66,7 +73,7 @@ def readinto(self, buf, *, start=0, end=None):
             end = len(buf)
         self.i2c.readfrom_into(self.device_address, buf, start=start, end=end)
 
-    def write(self, buf, *, start=0, end=None):
+    def write(self, buf: bytearray, *, start: int = 0, end: Optional[int] = None):
```

Comment:
This should use `ReadableBuffer` I think.

Reply:
It should, I think I based it off the docstring, so I'll change the type annotation and update the docstring as well.